aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc

```diff
@@ -48,22 +48,26 @@ image::{img}3desktopConnect_remote_graph.jpg[link="{img}3desktopConnect_remote_g
 image::{img}4desktopConnect_name_url.jpg[link="{img}4desktopConnect_name_url.jpg",role="popup-link"]
 
 [start=5]
-5. Now we need to choose an authentication method. For this example, we will use the standard username and password method. Choose that option.
+5. Now we need to choose an authentication method. For Neo4j Aura, we need to choose `Username/password`.
```

Comment:
select or pick for the 2nd choose ?

Reply:
Agreed, this isn't the best sentence.  I've updated to:
```
5. Now we need to choose an authentication method.  
Neo4j Aura instances require username and password authentication, so click the `Username/password` button.
```